### PR TITLE
Delete reserve function with confirm modal

### PR DESCRIPTION
**Pull request by yvallejos**
delete

### CODE REVIEW

File: app/components/reserves/delete/reserve.delete.controller.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  /* @ngInject */
+  function DeleteReserveController(ReservesService,$state,$uibModalInstance, reserve) {
+    var vm = this;
+
+    activate();
+
+    function activate() {
+    }
+
+    vm.deleteReserve = function () {
```
Comment (johnnyxu-pernix) [inline app/components/reserves/delete/reserve.delete.controller.js]: Create the function a then make the reference with vm. Like this:

``` 
vm.deleteReserve = deleteReserve;

function deleteReserve() {
/** Code here **/
}```

File: app/components/reserves/delete/reserve.delete.controller.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      ReservesService.deleteReserve(reserve)
+      .then(
+        function(){
+          $uibModalInstance.close();
+          $state.go('home.listReserves'); 
+        });
+    };
+
+    vm.cancel = function () {
```
Comment (johnnyxu-pernix) [inline app/components/reserves/delete/reserve.delete.controller.js]: Same here

File: app/components/reserves/delete/reserves.delete.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<div>
+  <div class="modal-header">
+    <h3 class="modal-title">Eliminar Reservación</h3>
+  </div>
+  <div class="modal-body">
+    Esta seguro de eliminar la reservación?
```
Comment (johnnyxu-pernix) [inline app/components/reserves/delete/reserves.delete.html]: Typo, `¿` missing here

File: app/components/reserves/show/reserve.show.controller.js [hunk 4]
```diff
@@ -6,7 +6,7 @@
     .controller('ShowReserveController', ShowReserveController);
 
   /* @ngInject */
-  function ShowReserveController(ReservesService,$state) {
+  function ShowReserveController(ReservesService,$state, $uibModal) {
```
Comment (johnnyxu-pernix) [inline app/components/reserves/show/reserve.show.controller.js]: Add some space between the parameters

File: app/components/reserves/show/reserve.show.controller.js [hunk 5]
```diff
@@ -30,11 +30,19 @@
     }
 
     function deleteReserve(){
+      var modalInstance = $uibModal.open({
```
Comment (johnnyxu-pernix) [inline app/components/reserves/show/reserve.show.controller.js]: Why are you creating the `var modalInstance`?

**Comment by johnnyxu-pernix** [comment]
@yvallejos maybe the `Si` button in the modal will look better if it were red. And maybe change the `Si` for `Eliminar` and the `No` for `Cancelar`.

File: app/components/reserves/delete/reserves.delete.html (hunk 3 above)
Comment (Emmurillo) [inline app/components/reserves/delete/reserves.delete.html]: "Está"

File: app/components/reserves/delete/reserve.delete.controller.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    .controller('DeleteReserveController', DeleteReserveController);
+
+  /* @ngInject */
+  function DeleteReserveController(ReservesService,$state,$uibModalInstance, reserve) {
+    var vm = this;
+
+    activate();
+
+    function activate() {
+    }
```
Comment (Emmurillo) [inline app/components/reserves/delete/reserve.delete.controller.js]: No need to add an activate function if this isn't doing nothing.

File: app/components/reserves/show/reserve.show.controller.js [hunk 7]
```diff
@@ -6,7 +6,7 @@
     .controller('ShowReserveController', ShowReserveController);
 
   /* @ngInject */
-  function ShowReserveController(ReservesService,$state) {
+  function ShowReserveController( ReservesService , $state , $uibModal ) {
```
Comment (Emmurillo) [inline app/components/reserves/show/reserve.show.controller.js]: @yvallejos The space is meant to be only after the coma.

**Comment by yvallejos** [comment]
delete

**Comment by Emmurillo** [comment]
@johnnyxl26, @joserv2812 this seems fine to me, merge it whether you guys agree.

File: app/components/reserves/delete/reserves.delete.html [hunk 8]
```diff
@@ -0,0 +1,12 @@
+<div>
```
Comment (Emmurillo) [inline app/components/reserves/delete/reserves.delete.html]: Just a nitpick: unnecessary div, just for the record

**Pull request merged**